client/src/workflows/Workflows.present.tsx

```diff
@@ -632,6 +607,45 @@ function VisualizerMappings({
   return (<>{elements}</>);
 }
 
+
+interface VisualizerLinksProps {
+  data: Record<string, any>[]
+  workflows: Record<string, any>;
+}
+
+function VisualizerLinks({
+  data, workflows
+}: VisualizerLinksProps) {
+  if (!data?.length)
+    return (<p className="m-2"><UnavailableDetail /></p>);
+
+  let num = 0;
+  let links = data.map((t: any) => {
```

Comment:
If you provide a second argument, you can get the index in map.
```suggestion
  let links = data.map((t: any, num: number) => {
  
  }
```

Reply:
Ah, thanks! I added it in a separate commit since I will also initialize it